e2e/ansible/playbooks/hyperconverged/test-k8s-memleak/memleak-cleanup.yml

```diff
@@ -8,8 +8,16 @@
   delegate_to: "{{groups['kubernetes-kubeminions'].0}}"
   register: result
 
+- name: Obtain the pvc name
+  shell: source ~/.profile; kubectl get pvc -n {{ namespace }} --no-headers | awk '{print $1}'
+  args:
+    executable: /bin/bash
+  register: pvc
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
```

Comment:
Better to derive the PV name, though this should work with the new conventions where PVC name is part of PV name.